src/plugins/timelist/inspector/svg-progress.js

```diff
@@ -0,0 +1,63 @@
+/************** https://marian-caikovski.medium.com/drawing-sectors-and-pie-charts-with-svg-paths-b99b5b6bf7bd */
```

Comment:
This blog post is paywalled. Is there a license associated with this code?

Reply:
I've removed the progress pie for now. We can add it back. Tracking the commit here: https://github.com/nasa/openmct/commit/fb38cdb60c227aa516c2c83ee9c0de0ab68ad59b